src/main/scala/com/codacy/client/stash/service/ProjectServices.scala

```diff
@@ -120,14 +122,4 @@ class ProjectServices(client: StashClient) {
         )()
       case None => client.executePaginated(Request(s"$BASE/$projectKey/permissions/groups", classOf[Seq[Group]]))()
     }
-
```

Comment:
Any specific reason for removing this endpoint? 

https://docs.atlassian.com/bitbucket-server/rest/7.0.1/bitbucket-rest.html#idp149